.github/workflows/tests.yml

```
@@ -0,0 +1,110 @@
+    - name: Create test environment file
+      run: |
+        cp .env.example .env
+        echo "FLASK_SECRET_KEY=test-secret-key" >> .env
+        echo "FLASK_DEBUG=False" >> .env
+    
+    - name: Run unit tests
+      run: |
+        pipenv run pytest tests/ -k "not e2e" --verbose
+    
+    - name: Run pylint
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:19: Quote this to prevent word splitting

(shellcheck)